packages/orchestration/src/facade.js

```diff
@@ -165,7 +169,20 @@ export const makeOrchestrationFacade = ({
     orchestrationService,
   });
 
+  const chainInfos = zone.mapStore('chainInfos', {
+    keyShape: M.string(),
+    // ??? when we support other chain types, will it be easy to change this shape?
```

Comment:
I doubt it.

Reply:
this didn't need to be durable so I changed this to a detached store and that makes it easy to change on upgrade